src/riak_kv_coverage_filter.erl

```diff
@@ -150,26 +154,26 @@ build_item_filter(FilterInput) ->
 
 
 %% @private
-build_preflist_fun(Bucket, CHBin) ->
+build_preflist_fun(Bucket, CHBin, KeyConvFn) ->
     fun({o, Key, _Value}) -> %% $ index return_body
-            ChashKey = riak_core_util:chash_key({Bucket, Key}),
+            ChashKey = riak_core_util:chash_key({Bucket, KeyConvFn(Key)}),
```

Comment:
Design was predicated on two things:

1. I didn't want the performance overhead of constructing the DDL _every time a key is evaluated_.  This is unnecessary, and doesn't scale.  Rather I wanted to construct a simple conversion function once, where the DDL was already available.

2. build_preflist_fun and build_subpartition_fun don't, and shouldn't IMO, know anything about details of the bucket, in particular whether or not it is a TS bucket.  As with the rest of the riak_core_coverage_fsm/riak_kv_coverage_filter code, I wanted to preserve as much as possible the model of doing TS-specific things in TS-specific code, and having the rest of the codepath be TS/KV-agnostic.  The more in-place checks for TS vs non-TS buckets and keys and encodings we pepper throughout the codebase, the more unmaintainable it becomes.